app/steps/ui/executors/additionalinvite/index.js

```diff
@@ -13,6 +13,7 @@ class ExecutorsAdditionalInvite extends ValidationStep {
 
     getContextData(req) {
         const ctx = super.getContextData(req);
+        ctx.executorsToNotifyList = new ExecutorsWrapper(ctx).executorsToNotify();
```

Comment:
Isn't this redundant?
`executorsToNotifyList` should already be in `session.form.data.executors`, therefore `const ctx = super.getContextData(req);` should add it to `ctx`.